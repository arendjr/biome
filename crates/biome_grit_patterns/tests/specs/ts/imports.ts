import lib from "library";
import * as lib from "library";
import lib, { util as combinedUtil } from "library";
import { util } from "library";
import { util as namedUtil } from "library";

import type lib2 from "library";
import { type util2 } from "library";
import { type util2 as namedUtil2 } from "library";
