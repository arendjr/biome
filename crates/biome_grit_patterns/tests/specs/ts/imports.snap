---
source: crates/biome_grit_patterns/tests/spec_tests.rs
expression: imports
---
SnapshotResult {
    messages: [],
    matched_ranges: [
        "1:1-1:27",
        "2:1-2:32",
        "3:1-3:53",
        "4:1-4:32",
        "5:1-5:45",
        "7:1-7:33",
        "8:1-8:38",
        "9:1-9:52",
    ],
    rewritten_files: [],
    created_files: [],
}
